internal/datacoord/server.go

```diff
@@ -773,6 +773,9 @@ func (s *Server) loadCollectionFromRootCoord(ctx context.Context, collectionID i
 // GetVChanPositions get vchannel latest postitions with provided dml channel names
 func (s *Server) GetVChanPositions(channel string, collectionID UniqueID, seekFromStartPosition bool) *datapb.VchannelInfo {
 	segments := s.meta.GetSegmentsByChannel(channel)
+	log.Debug("GetSegmentsByChannel",
+		zap.Any("segments", segments),
+	)
 	flushed := make([]*datapb.SegmentInfo, 0)
```

Comment:
Printing all segments info will contains binlog paths. This may be annoying when looking into log.

Reply:
fixed